src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  font-family: "Avenir-Book", Arial, Helvetica, sans-serif;
+}
+
+.input {
+  display: flex;
+
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+
+  background: url(./images/Search.svg) no-repeat center left 25px;
```

Comment:
use pseudoelement or tag `img` to prevent code duplicating

Reply:
ok,
I`ve removed  background: url() and used img instead with the position: absolute